sunpy/instr/aia/response.py

```diff
@@ -46,20 +46,19 @@ def get_function(ion, emissivity, temperature, density, optional = None):
     pass
 # think: how to test
 
-def read_genx(path_to_file):
+def get_data_from_aia_inst_file(path_to_file):
```

Comment:
still, this will give you the idea, as a user, that you will get that in your session, not that then you'll have to read again from a different file. I could suggest, if you still want to read it and save it as csv to call it something like:
`aia_inst_genx2csv`